src/js/handlers/handleCalClicked.js

```diff
@@ -0,0 +1,58 @@
+import {
+  INITIAL_VALUE,
+  ERROR_MESSAGES,
+  OPERATORS,
+} from "../utils/constants.js";
+import { TOTAL, DIGITS, OPERATIONS, ALLCLEAR } from "../utils/DOM.js";
+import { isAbleOperator, isValidDigitLength } from "./validator.js";
+import { operation } from "../utils/operation.js";
+
+const displayDigit = (digit) => {
+  if (!isValidDigitLength(TOTAL)) {
+    alert(`${ERROR_MESSAGES.INVALID_DIGIT_LENGTH}`);
+    return;
+  }
+  if (TOTAL.textContent === INITIAL_VALUE) {
+    TOTAL.textContent = digit;
+  } else {
+    TOTAL.textContent += digit;
+  }
```

Comment:
```suggestion
  TOTAL.textContent = Number(TOTAL.textContent + digit);
```
이렇게 표현해주면 어떨까요?
![image](https://user-images.githubusercontent.com/18749057/177332004-d9322277-0012-4b33-a72f-110736871a10.png)